kalite/settings.py

```diff
@@ -170,7 +170,6 @@ def package_selected(package_name):
 import_installed_app_settings(INSTALLED_APPS, globals())
 
 # Override
-KHAN_EXERCISES_DIRPATH = getattr(local_settings, "KHAN_EXERCISES_DIRPATH", os.path.join(STATIC_ROOT, "khan-exercises"))
```

Comment:
Move the definition from distributed/settings.py, then wrap it in the `getattr(local_settings, "KHAN_EXERCISES_DIRPATH", ...)` call, to allow users to override it.